src/codeflare_sdk/utils/demos.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+import shutil
+
+package_dir = pathlib.Path(__file__).parent.parent.resolve()
+demo_dir = f"{package_dir}/demo-notebooks"
+
+
+def copy_demo_nbs(dir: str = "./demo-notebooks", overwrite: bool = False):
+    """
+    Copy the demo notebooks from the package to the current working directory
+
+    overwrite=True will overwrite any files that exactly match files written by clone_demo_nbs in the target directory.
```

Comment:
overwrite=True will overwrite any files that exactly match files written by copy_demo_nbs in the target directory.